pkg/utils/controller/metrics/metrics.go

```diff
@@ -54,6 +54,13 @@ var (
 		Name: "fleet_workload_placement_complete",
 		Help: "Placement complete status ",
 	}, []string{"name"})
+
+	// FleetEvictionStatus is prometheus metrics which holds the
+	// status of eviction completion.
+	FleetEvictionStatus = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "fleet_workload_eviction_complete",
+		Help: "Eviction complete status ",
+	}, []string{"name", "isCompleted"})
```

Comment:
I think it will be very helpful to have another label about if it's a user error